Uplift/Views/GymDetail/GymDetailTimeInfoView.swift

```diff
@@ -128,6 +152,28 @@ class GymDetailTimeInfoView: UIView {
 
         return ""
     }
+
+    func setupEmptyState() {
+        emptyStateView = NoHoursEmptyStateView()
+        addSubview(emptyStateView)
+
+        emptyStateView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+
+    func getTimesHeight(for hours: [FacilityHoursRange]) -> CGFloat {
+
+        let emptyStateHeight = CGFloat(163)
+
+        let lineHeight: CGFloat = 19.6
+        let lineSpacing: CGFloat = 5.0
+        let hoursCount = CGFloat(hours.count)
+
+        let timesHeight = (hoursCount == 1 && hours[0].openTime == hours[0].closeTime) ? emptyStateHeight : lineHeight * hoursCount + lineSpacing * (hoursCount - 1)
```

Comment:
can you put `(hoursCount == 1 && hours[0].openTime == hours[0].closeTime)` into a variable so its clear what it means